.github/workflows/ci_linux.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         #
         #         apt-get install python3-venv
         #
-        sudo apt -y install libsdl2-dev python3-venv valgrind spirv-tools glslang-dev libvulkan-dev mesa-vulkan-drivers vulkan-validationlayers
+        sudo apt -y install libsdl2-dev libva-dev python3-venv valgrind spirv-tools glslang-dev libvulkan-dev mesa-vulkan-drivers vulkan-validationlayers
```

Comment:
can you add it to coverage as well for consistency?

Reply:
Done.